go.mod

```diff
@@ -94,7 +94,7 @@ replace (
 	k8s.io/kube-scheduler => github.com/openshift/kubernetes/staging/src/k8s.io/kube-scheduler v0.0.0-20210429023751-daa555af5040
 	k8s.io/kubectl => github.com/openshift/kubernetes/staging/src/k8s.io/kubectl v0.0.0-20210429023751-daa555af5040
 	k8s.io/kubelet => github.com/openshift/kubernetes/staging/src/k8s.io/kubelet v0.0.0-20210429023751-daa555af5040
-	k8s.io/kubernetes => github.com/openshift/kubernetes v1.21.0-rc.0.0.20210426183659-cd6d8b086ad5
+	k8s.io/kubernetes => github.com/openshift/kubernetes v1.21.0-rc.0.0.20210518181655-9bec1e131ca5
 	k8s.io/legacy-cloud-providers => github.com/openshift/kubernetes/staging/src/k8s.io/legacy-cloud-providers v0.0.0-20210429023751-daa555af5040
```

Comment:
I think we might want to bump all of these openshift/kubernetes inclusions with:

```console
$ hack/update-kube-vendor.sh 9bec1e131ca503a7f9850ef321e073b121fa7c79
```

to avoid melting our brains by mixing a composite of openshift/kubernetes commits in a single origin build.  But I'm not an origin maintainer, and if they prefer this narrower pivot, that's fine with me.

Reply:
I've opened #26169 with the `update-kube-vendor.sh` approach, in case the admins prefer that, although I'm fine either way.  Once they decide, we can just close the one that didn't merge and drop it from the bug.